lib/dispatch-rider/scheduled_job/migration.rb

```diff
@@ -0,0 +1,13 @@
+module DispatchRider
+  module ScheduledJob::Migration
+    def create_scheduled_jobs_table
+      create_table "scheduled_jobs" do |t|
+        t.datetime "scheduled_at"
+        t.text   "destinations"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.